examples/sql-import/.lando.yml

```diff
@@ -3,7 +3,10 @@ events:
   post-db-import:
     - database: /app/test-script.sh
     - database: echo "i-ran" > /tmp/iran.txt
+    - echo "stuff"
 services:
+  appserver:
+    type: mysql
```

Comment:
Should this be apache or nginx instead?  Seems that would be the more likely appserver service?

Reply:
@uberhacker the underlying service doesnt matter, the point is to just have any service called `appserver` and reusing an image thats already been pulled is the most efficient way to do that